packages/mwp-logger-plugin/src/logger.js

```diff
@@ -44,7 +44,7 @@ export const httpRequestSerializers = {
 			requestInfo.responseSize = request.response.headers['content-length'];
 			requestInfo.status = request.response.statusCode;
 			requestInfo.latency = formatDuration(
-				request.info.responded - request.info.received
+				request.info.completed - request.info.received
```

Comment:
In Hapi 18, `request.info.responded` is a  non-0 value (timestamp) only if the response was successfully sent.

> If you access request.info.responded (usually in logging logic), consider switching to request.info.completed if you want to know when the server finished processing and use the existence of request.info.responded to log if a response was successful.